packages/ui-extensions-server-kit/src/testing/extensions.ts

```diff
@@ -50,10 +50,7 @@ export function mockExtension(obj: DeepPartial<ExtensionPayload> = {}): Extensio
     // [see stackoverflow](https://stackoverflow.com/a/68699273) assume that
     // `DeepPartial<Array<T>> === Array<T | undefined>` while we are looking for
     // `DeepPartial<Array<T>> === Array<T> | undefined`.
-    // This is the case for extension points and also categories and seems hard to fix
```

Comment:
You may want to leave this comment, but just remove the reference to `categories` since the comment still applies to `extensionPoints`, I think